packages/crawler/src/page-processors/page-processor-helper.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import Apify from 'apify';
+import { Logger } from 'logger';
+import { Page } from 'puppeteer';
+import { ScanData } from '../scan-data';
+import { BlobStore } from '../storage/store-types';
+
+export type PartialScanData = {
+    url: string;
+    id: string;
+} & Partial<ScanData>;
+
+export class PageProcessorHelper {
```

Comment:
why we need helper class when two functions below are used only from processors and were part of abstract base class?

Reply:
Since both page processors (classic and simulator) use these methods, I thought it would be easier to move them into a separate object that we can easily mock for testing, instead of testing for the behavior of base class methods (which would mean changing multiple tests if the base class changes)